src/crsAdapter/BewotecExpertAdapter.js

```diff
@@ -548,8 +549,8 @@ class BewotecExpertAdapter {
                 },
                 {}
             )[traveller.salutation],
-            firstName: travellerName[0],
-            lastName: travellerName[1],
+            firstName: travellerName.filter(Boolean).join(' '),
```

Comment:
i guess you could leave out the `.filter(Boolean)` because at this point you will always have an array of strings - at least `[""]`